maat-court-data-api/src/test/java/gov/uk/courtdata/link/service/CreateLinkListenerTest.java

```diff
@@ -32,7 +35,7 @@ public void givenJSONMessageIsReceived_whenCreateLinkListenerIsInvoked_thenCreat
         String message = "Test JSON";
         //when
         when(gson.fromJson(message, CaseDetails.class)).thenReturn(caseDetails);
-        createLinkListener.receive(message, "1");
+        createLinkListener.receive(message, new MessageHeaders(new HashMap<>()));
```

Comment:
Same here we only test the happy path so lets add a test case that fails the validation checks.